app/Main.hs

```diff
@@ -40,6 +40,8 @@ import Text.PrettyPrint.ANSI.Leijen.Internal (Doc, linebreak)
 import qualified Lorentz.Contracts.Oracle.CmdLnArgs as OracleCmdLnArgs
 -- import qualified Lorentz.Contracts.Admin.Simple as SimpleAdmin
 import qualified Lorentz.Contracts.Admin.Simple.CmdLnArgs as SimpleAdminCmdLnArgs
+-- import qualified Lorentz.Contracts.Admin.Signed as SignedAdmin
+import qualified Lorentz.Contracts.Admin.Simple.CmdLnArgs as SignedAdminCmdLnArgs
```

Comment:
This is the same import as line 42: https://github.com/tqtezos/lorentz-contract-oracle/pull/2/files#diff-63e7d2cbf23880a9c902c91fa34f6043R42

Reply:
I.e., it's the `SimpleAdmin` CLI interface.